roles/openshift_aws/tasks/dns_route53_record.yml

```diff
@@ -0,0 +1,25 @@
+---
+- debug: msg="{{ openshift_aws_dns_records[l_openshift_aws_dns_element] }}"
+
+- name: querying elb
+  ec2_elb_facts:
+    names: "{{ openshift_aws_dns_records[l_openshift_aws_dns_element]['value'] }}"
+    region: "{{ openshift_aws_region }}"
+  register: elb_facts
+  when:
+    - "'elb' in openshift_aws_dns_records[l_openshift_aws_dns_element]['kind']"
+    - "'CNAME' in openshift_aws_dns_records[l_openshift_aws_dns_element]['type']"
+
+- name: creating record
```

Comment:
any looping necessary should appear right here; We can utilize with_dict on this task to iterate through the dictionary of records.

Reply:
Keep in mind we have to discover the dns name of the elb.  elb names don't work.  we have to loop two tasks and looping blocks don't work.